package.json

```diff
@@ -29,7 +30,8 @@
     "fastify": "^4.14.0",
     "fastify-plugin": "^4.5.0",
     "mongoose": "^7.0.5",
-    "pino": "^8.11.0"
+    "pino": "^8.11.0",
+    "uuid4": "^2.0.3"
```

Comment:
Where is `uuid4` used? 

If we do, in fact, use it, let's replace it with `randomUuid4()` from `node:crypto`.